catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ValidateCommand.java

```diff
@@ -47,178 +55,104 @@
 @Service
 public class ValidateCommand implements Action {
 
-    @Argument(index = 0, name = "fileName", description = "The path to the file that you want to validate", required = true, multiValued = false)
+    @Option(name = "--path", aliases = "-p", description = "The path to the file to be validated")
```

Comment:
Ignore me. Jeez. The `Option` annotation you're using is _not_ deprecated. I thought you'd replaced with the old one. You're fine. 👍